include/libopencm3/stm32/h7/pwr.h

```diff
@@ -55,32 +55,80 @@ LGPL License Terms @ref lgpl_license
 /** Wakeup Domain Power Control register. */
 #define PWR_WKUPCR      MMIO32(POWER_CONTROL_BASE + 0x20)
 
+/** SYSCFG Register for configuring regulator "overdrive" */
+
 /*@}*/
 
 /** VOS[15:14]: Regulator voltage scaling output selection */
 #define PWR_CR1_SVOS_SHIFT        14
 #define PWR_CR1_SVOS_SCALE_3      (0x3)
 #define PWR_CR1_SVOS_SCALE_4      (0x2)
 #define PWR_CR1_SVOS_SCALE_5      (0x1)
+
 #define PWR_CR1_SVOS_MASK         (0x3)
 
 /** DBP[8]: Disable backup domain write protection. */
 #define PWR_CR1_DBP               (1 << 8)
 
-/** PVDO: PVD output */
-#define PWR_CSR1_PVDO             (1 << 4)
+/** CSR1 Register Bits */
+#define PWR_CSR1_AVDO             BIT16
+#define PWR_CSR1_ACTVOS_SHIFT     14
+#define PWR_CSR1_ACTVOSRDY        BIT13
+#define PWR_CSR1_PVDO             BIT24
+
+/** CR3 Register Bits */
+#define PWR_CR3_USB33RDY          BIT26
+#define PWR_CR3_USBREGEN          BIT25
+#define PWR_CR3_USB33DEN          BIT24
+#define PWR_CR3_VBRS              BIT9
+#define PWR_CR3_VBE               BIT8
+#define PWR_CR3_SCUEN             BIT2
+#define PWR_CR3_LDOEN             BIT1
+#define PWR_CR3_BYPASS            BIT0
+
+/** D3CR Register Bits */
+#define PWR_D3CR_VOS_SHIFT        14
+#define PWR_D3CR_VOSRDY           BIT13
+
+#define PWR_D3CR_VOS_SCALE_3      (0x3)
+#define PWR_D3CR_VOS_SCALE_2      (0x2)
+#define PWR_D3CR_VOS_SCALE_1      (0x1)
+#define PWR_D3CR_VOS_MASK         (0x03)
 
 /* --- Function prototypes ------------------------------------------------- */
 enum pwr_svos_scale {
-  PWR_SCALE3 = PWR_CR1_SVOS_SCALE_3 << PWR_CR1_SVOS_SHIFT,
-  PWR_SCALE4 = PWR_CR1_SVOS_SCALE_4 << PWR_CR1_SVOS_SHIFT,
-  PWR_SCALE5 = PWR_CR1_SVOS_SCALE_5 << PWR_CR1_SVOS_SHIFT,
+  PWR_SVOS_SCALE3 = PWR_CR1_SVOS_SCALE_3 << PWR_CR1_SVOS_SHIFT,
+  PWR_SVOS_SCALE4 = PWR_CR1_SVOS_SCALE_4 << PWR_CR1_SVOS_SHIFT,
+  PWR_SVOS_SCALE5 = PWR_CR1_SVOS_SCALE_5 << PWR_CR1_SVOS_SHIFT,
+};
+
+enum pwr_vos_scale {
+  PWR_VOS_SCALE_0 = 0,    /* Note: This state requires SYSCFG ODEN set. */
```

Comment:
As we get to later, I'm not really sure you should have this case specially

Reply:
Again, this is explicitly referenced in the docs.